core/src/main/scala/kafka/server/FetchSession.scala

```diff
@@ -781,11 +781,7 @@ class FetchManager(private val time: Time,
                 cache.remove(session)
                 new SessionlessFetchContext(fetchData)
               } else {
-                if (session.size != session.cachedSize) {
-                  // If the number of partitions in the session changed, update the session's
-                  // position in the cache.
-                  cache.touch(session, session.lastUsedMs)
```

Comment:
Currently supplies the session.lastUsedMs which means that lastUsedMs will be set to the current lastUsedMs.